src/pcapi/core/users/models.py

```diff
@@ -273,3 +307,16 @@ def needsToSeeTutorials(self):
     @property
     def hasOffers(self):
         return any([offerer.nOffers > 0 for offerer in self.offerers])
+
+
+class ExpenseDomain(enum.Enum):
+    ALL = "all"
+    DIGITAL = "digital"
+    PHYSICAL = "physical"
+
+
+@dataclass
+class Expense:
+    domain: ExpenseDomain
+    current: Union[Decimal, int]
```

Comment:
Du coup, je pense que l'on peut:

```suggestion
    current: Decimal
```